vsg/rules/generate/rule_011.py

```diff
@@ -126,3 +160,15 @@ def manage_labels(oToken, lLabels):
         return True
 
     return False
+
+
+def remove_keyword(self):
+    if self.action == 'remove':
+        return True
+    return False
+
+
+def create_violation(oToi, iLineNumber, self):
+    sSolution = self.action.capitalize() + ' ' + self.solution
```

Comment:
Because self.solution is "Add generate label", if the action is `remove`, `sSolution` here becomes "Remove Add generate label". Perhaps the solution is to change `self.solution` in `init()` to "generate label"? `

Reply:
@JHertz5 , you are correct.  setting self.solution to "generate label" fixes the issue.  I pushed an updated to the branch.  Let me know if there are any other issues.

--Jeremy

Reply:
I can confirm that the update has fixed the issue.